pkgs/development/tools/gocyclo/default.nix

```diff
@@ -1,26 +1,24 @@
-{ buildGoPackage
-, lib
-, fetchFromGitHub
-}:
+{ buildGoPackage, lib, fetchFromGitHub }:
 
 buildGoPackage rec {
-  pname = "gocyclo-unstable";
-  version = "2015-02-08";
-  rev = "aa8f8b160214d8dfccfe3e17e578dd0fcc6fede7";
+  pname = "gocyclo";
+  version = "0.3.1";
+  rev = "v${version}";
 
-  goPackagePath = "github.com/alecthomas/gocyclo";
+  goPackagePath = "github.com/fzipp/gocyclo";
 
   src = fetchFromGitHub {
     inherit rev;
 
-    owner = "alecthomas";
+    owner = "fzipp";
     repo = "gocyclo";
-    sha256 = "094rj97q38j53lmn2scshrg8kws8c542yq5apih1ahm9wdkv8pxr";
+    sha256 = "1rimdrhmy6nkzh7ydpx6139hh9ml6rdqg5gvkpy2l1x9mhanvan0";
   };
 
   meta = with lib; {
-    description = "Calculate cyclomatic complexities of functions in Go source code";
-    homepage = "https://github.com/alecthomas/gocyclo";
+    description =
+      "Calculate cyclomatic complexities of functions in Go source code";
+    homepage = "https://github.com/fzipp/gocyclo";
```

Comment:
Please link to some statement that legitimized the change of source url here for future reference. I like to document that we're not slipping malicious software into nixpkgs when possible.

Reply:
There's not really a statement for this, if you look at https://github.com/alecthomas/gocyclo, you'll see it's forked from https://github.com/fzipp/gocyclo, which is _vastly_ more up-to-date.

Perhaps just a note in the commit message?

Reply:
So the project hasn't been transferred... just that a random developer forked it. Worth briefly mentioning that we're pulling sources from another authot now.